Cannot find module 'rsvp' at installation

I tried to install the addon on an Ember app using ember-cli v2.8.0 and the above error was given and the installation was interrupted.

ENV:
node: 4.5.0 LTS
npm: 2.15.9
ember-cli: 2.8.0

Recreate:
With the above environment do `ember install ember-cli-chart`
That gives the following output: (app path is replaced with `<app>`)

```
Installed packages for tooling via npm.
Cannot find module 'rsvp'
Error: Cannot find module 'rsvp'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/<app>/node_modules/ember-cli-chart/blueprints/ember-cli-chart/index.js:1:14)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
```

--- a/package.json
+++ b/package.json
@@ -49,7 +49,8 @@
     "ember-load-initializers": "^0.5.1",
     "ember-resolver": "^2.0.3",
     "ember-try": "^0.2.2",
-    "loader.js": "^4.0.1"
+    "loader.js": "^4.0.1",
+    "rsvp": "^3.3.2"
   },
   "dependencies": {
     "ember-cli-babel": "^5.1.6"

